pom.xml

```diff
@@ -1210,6 +1206,28 @@
 
     <plugins>
       <plugin>
+        <groupId>org.scijava</groupId>
+        <artifactId>scijava-maven-plugin</artifactId>
```

Comment:
The license of sci is a bit weird but we just use it (not embedding) so it's OK.